internal/app/app.go

```diff
@@ -63,6 +62,9 @@ type App struct {
 	// Type is the type of application this is
 	Type Type
 
+	// Abstraction over Custom or Bootstrap deployment
```

Comment:
nit: Should start with the name of the field

```suggestion
	// Deployment is an abstraction over Custom or Bootstrap deployment
```